nestkernel/eprop_archiving_node.h

```diff
@@ -23,6 +23,9 @@
 #ifndef EPROP_ARCHIVING_NODE_H
 #define EPROP_ARCHIVING_NODE_H
 
+// C++ includes:
```

Comment:
```suggestion
// C++
```
In the other files we do not repeat "include" in the title of a new list of includes. It's clear that its a list of includes, since the lines start with `#include`